features/draft_environment.feature

```diff
@@ -32,9 +32,3 @@ Feature: Draft environment
     Then I should see "Content design"
     And I should see "Search this manual"
     And the page should contain the draft watermark
-
-  @draft
-  Scenario: visiting a page served by contacts-frontend
```

Comment:
I read the tests in that file to be testing rendering apps via the draft stack - there is already a `government-frontend` test in there. There are other features for testing general rendering:

* https://github.com/alphagov/smokey/blob/master/features/contacts.feature
* https://github.com/alphagov/smokey/blob/master/features/government_frontend.feature